examples/_d2n-std/nix/app/packages.nix

```diff
@@ -0,0 +1,22 @@
+{
+  inputs,
+  cell,
+}: {
+  default = cell.packages.app;
+  app =
+    # a terrible idea
```

Comment:
This could work but completely destroys the harmony :smiley: 

But I don't think this is the right way. There must be a better way.

Reply:
Currently, we do not have a less terrible interface ;)
I plan to re-design this completely
I recently created a collaborative document to discuss how the new drem2nix API should look like.
Feel free to collaborate and dump your ideas there: https://pad.lassul.us/UswFgE39SDuvRcT3R7irNQ?view

To summarize it. I plan to change dream2nix to not provide any `functions` itself, and instead only export modules for drv-parts. After this, everything will evolve around individual derivations instead of flakes. Composing multiple packages to something like a flake will then be up to the user.

Reply:
Ok, that is frankly extremely promising.

I engaged in a round of (extensive) comments trying to fully cover my stakeholder perspective as a:

- DevOps persona operating on the intersection of Ops, Build & DevX
- Horizontal Integration Framework maintainer (i.e. `std`)